platform/nodejs/tests/snfe.js

```diff
@@ -34,78 +34,10 @@ import {
 let engine = null;
 
 describe('SNFE', () => {
-    function fetch(listName) {
```

Comment:
All of this code was copied from `test.js`. Now it runs as part of `test.js` anyway.